Tests/NumLib/TestShapeFunctions.cpp

```diff
@@ -11,16 +11,142 @@
 
 #include <gtest/gtest.h>
 
+#include <algorithm>
+#include <type_traits>
 #include <limits>
+#include <numeric>
 #include <valarray>
-#include <algorithm>
 
+#include "MeshLib/Elements/Elements.h"
+#include "NumLib/Fem/ShapeFunction/ShapeHex20.h"
+#include "NumLib/Fem/ShapeFunction/ShapeHex8.h"
+#include "NumLib/Fem/ShapeFunction/ShapeLine2.h"
+#include "NumLib/Fem/ShapeFunction/ShapeLine3.h"
+#include "NumLib/Fem/ShapeFunction/ShapePrism15.h"
```

Comment:
I don't see all types included here in the `ShapeFunctionTestTypes` in line 101 ff. Why are there more included types (`ShapePrism*`, `ShapePyra*`, `ShapeQuad9`)?